.ado/templates/apple-xcode-build-static-libs.yml

```diff
@@ -0,0 +1,63 @@
+#
+# Build the FluentUI React Native Test App in different configurations
+#
+steps:
+  # macOS Framework Debug
+  - template: apple-xcode-build.yml
+    parameters:
+      xcode_sdk: 'macosx'
+      xcode_workspacePath: 'apps/macos/src/FluentUITester.xcworkspace'
+      xcode_actions: 'build'
+      xcode_scheme: 'ReactTestApp'
+      xcode_configuration: 'Debug'
+      xcode_extraArgs: '-xcconfig $(Build.Repository.LocalPath)/.ado/xcconfig/publish_overrides.xcconfig'
+
+  # macOS Framework Release
+  - template: apple-xcode-build.yml
+    parameters:
+      xcode_sdk: 'macosx'
+      xcode_workspacePath: 'apps/macos/src/FluentUITester.xcworkspace'
+      xcode_actions: 'build'
+      xcode_scheme: 'ReactTestApp'
```

Comment:
`ReactTestApp`? What about FURNTestApp or something that has to do with FluentUI

Reply:
ReactTestApp is the scheme from ''react-native-test-app', I don't think we can control that name. It would be best if we had a scene that was just a status library, but I don't think 'react-native-test-app' had support for that

Reply:
Ah, we're using `react-native-test-app`- forgot about that. I think it's fine as is then :)

Reply:
Sounds like a bug in react-native-test-app? We should at least create an issue?

Reply:
Not so much a bug but a feature ask? I'll create an issue in their repo though

Reply:
Filed https://github.com/microsoft/react-native-test-app/issues/239

Reply:
https://github.com/microsoft/react-native-test-app/issues/239